FeedSX/screens/postDetail/index.tsx

```diff
@@ -703,8 +861,76 @@ const PostDetail = (props: IProps) => {
             </TouchableOpacity>
           </View>
         )}
-        {/* text input section for comment */}
+        {/* users tagging list */}
+        {allTags && isUserTagging ? (
+          <View
+            style={[
+              styles.taggingListView,
+              {
+                paddingBottom: replyOnComment.textInputFocus
+                  ? Layout.normalize(74)
+                  : Layout.normalize(44),
+                height: userTaggingListHeight,
+              },
+            ]}>
+            <FlashList
+              data={[...allTags]}
+              renderItem={({item}: any) => {
```

Comment:
rename `item` to `user`

Reply:
item is used as standard name in flashlist